src/main/java/org/folio/innreach/client/RequestStorageClient.java

```diff
@@ -2,16 +2,22 @@
 
 import java.util.UUID;
 
+import org.folio.innreach.domain.dto.folio.requeststorage.RequestDTO;
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 
 import org.folio.innreach.config.FolioFeignClientConfig;
 import org.folio.innreach.domain.dto.folio.requeststorage.RequestsDTO;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 
 @FeignClient(name = "request-storage", configuration = FolioFeignClientConfig.class)
 public interface RequestStorageClient {
 
   @GetMapping("/requests?query=(itemId==\"{itemId}\")")
   RequestsDTO findRequests(@PathVariable("itemId") UUID itemId);
```

Comment:
Consider to use `org.folio.innreach.domain.dto.folio.ResultList`. FOLIO endpoints that return collections usually have common structure, ResultList can be used in such cases